### ISSUE TITLE
Pb géolocalisation

### ISSUE THREAD

**Issue by bengrol** [Shuunen/bergerac-roads#48]
Je n'arrive pas à me géolocaliser en web.

**Comment by Shuunen** [comment]
navigateur ?
version ? 
système ?
32 / 64 bit ?

**Comment by Shuunen** [comment]
je viens de le faire sous Windows 10 64 bit avec Chrome dernière version

**Comment by bengrol** [comment]
En fait ça fonctionne sue Web 
( Sur Chrome - Linux - Version 64.0.3282.186 (Official Build) (64-bit) ) 

Mais sur iphone 6 (Safari ) il ne me demande pas si j'accepte de partager ma localisation et il tourne en boucle.

**Comment by Shuunen** [comment]
![iphone6 demo](https://user-images.githubusercontent.com/439158/44871566-72250980-ac93-11e8-96a1-584789213296.gif)

je viens de tester ça @bengrol

**Comment by bengrol** [comment]
Je n'ai pas cette fenêtre qui s'affiche.


Le jeu. 30 août 2018 à 20:30, Romain Racamier <notifications@github.com> a
écrit :

**Comment by Shuunen** [comment]
yes ça doit être autorisé pour cette API key (sur la console de management Google) :

Geocoding Service: This API project is not authorized to use this API.  For more information on authentication and Google Maps JavaScript API services please see: https://developers.google.com/maps/documentation/javascript/get-api-key

**Comment by Shuunen** [comment]
@bengrol il nous faut aussi ce service : 

Directions Service: This API project is not authorized to use this API. 

Sinon on ne peut ni localiser ni faire d'itinéraires, peut importe la plateforme
